src/libraries/System.Private.Uri/src/System/UncNameHelper.cs

```diff
@@ -49,7 +37,7 @@ internal static string ParseCanonicalName(string str, int start, int end, ref bo
         //
         // Assumption is the caller will check on the resulting name length
         // Remarks:  MUST NOT be used unless all input indexes are verified and trusted.
-        internal static unsafe bool IsValid(char* name, ushort start, ref int returnedEnd, bool notImplicitFile)
+        public static unsafe bool IsValid(char* name, ushort start, ref int returnedEnd, bool notImplicitFile)
```

Comment:
This is the only one I'm not sure of; I've seen this inconsistency elsewhere and not sure if we have a recommendation.

Reply:
>not sure if we have a recommendation.

It might not be written down but the recommendation is to use 'internal' for classes that need to be used throughout the assembly (i.e. System.Private.Uri) but then use only 'public' or 'private' for methods, properties, etc.